schema.graphql

```diff
@@ -23,28 +24,36 @@ type VerseReference {
 type BibleVersion {
   id: ID!
   name: String! @search(by: [term, exact])
-  language: ISOLanguage! 
-  script: ISOScript!
+  language: ISOLanguage! @hasInverse(field: bibleVersions)
+  script: ISOScript! @hasInverse(field: bibleVersions)
   abbreviation: String @search @id
   rights: String @search(by: [term, exact])
+  localizedBooks: [LocalizedBookName] @hasInverse(field: bibleVersion)
   revisions: [BibleRevision] @hasInverse(field: bibleVersion)
   forwardTranslation: BibleRevision @hasInverse(field: backTranslation)
   backTranslation: BibleRevision @hasInverse(field: forwardTranslation)
   machineTranslation: Boolean
 }
 
+type LocalizedBookName {
+  id: ID!
+  name: String! @search(by: [term, exact])
+  book: BookReference! @hasInverse(field: localizations)
+  bibleVersion: BibleVersion! @search(by: [exact]) @hasInverse(field: localizedBooks)
+}
+
 type ISOLanguage {
   id: ID!
   iso639: String! @search(by: [exact]) @id
   name: String! @search(by: [term, exact])
-  bibleVersions: [BibleVersion]
+  bibleVersions: [BibleVersion] @hasInverse(field: language)
 }
 
 type ISOScript {
   id: ID!
   iso15924: String! @search(by: [exact]) @id
   name: String! @search(by: [term, exact])
-  bibleVersions: [BibleVersion]
+  bibleVersions: [BibleVersion] @hasInverse(field: script)
```

Comment:
`isoScript`?

Reply:
This and the above are all done. I can't approve my own PR, but it should be ready to be merged.